src/bindgen/library.rs

```diff
@@ -42,7 +45,22 @@ impl Library {
         typedefs: ItemMap<Typedef>,
         functions: Vec<Function>,
         source_files: Vec<PathBuf>,
+        assoc_types: HashMap<AssocTypeId, (Type, u8)>,
```

Comment:
```suggestion
        assoc_types: HashMap<AssocTypeId, Type>,
```

Move this conversion of `filtered_assoc` to the call site